### PR TITLE
Fix "Error: No such container" on buildAndRun shell-scripts

### PR DESCRIPTION
**Pull request by tgotwig**
Relates #2

### CODE REVIEW

**Comment by rieckpil** [comment]
thanks for the contribution @TGotwig!

When I run the script for a fresh project I still get the error:

```
➜  jsf-demop ./buildAndRun.sh
Error: No such container: jsf-demop
1f6041aa4349f3414d18f6965eac363b0726d33808a4fb5f617cf966beb9a08f
➜  jsf-demop cat buildAndRun.sh 
#!/bin/sh
if [ "$(docker ps -a -f name=jsf-demop | grep -w jsf-demop | wc -l)" -eq 1 ]
then
  mvn clean package && docker build -t de.rieckpil.blog/jsf-demop .
fi
docker rm -f jsf-demop || true && docker run -d -p 9080:9080 -p 9443:9443 --name jsf-demop de.rieckpil.blog/jsf-demop
```

File: jakartaee-8-archetype/src/main/resources/archetype-resources/buildAndRun.sh [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
-mvn clean package && docker build -t ${groupId}/${artifactId} .
-docker rm -f ${artifactId} || true && docker run -d -p 9080:9080 -p 9443:9443 --name ${artifactId} ${groupId}/${artifactId}
\ No newline at end of file
+if [ "$(docker ps -a -f name=${artifactId} | grep -w ${artifactId} | wc -l)" -eq 1 ]
+then
+  mvn clean package && docker build -t ${groupId}/${artifactId} .
+fi
+docker rm -f ${artifactId} || true && docker run -d -p 9080:9080 -p 9443:9443 --name ${artifactId} ${groupId}/${artifactId}
```
Comment (rieckpil) [inline jakartaee-8-archetype/src/main/resources/archetype-resources/buildAndRun.sh]: shouldn't only `docker rm -f ${artifactId} ` be inside the if? To only run `docker rm` if we find an existing container with the same name?

Comment (tgotwig) [reply jakartaee-8-archetype/src/main/resources/archetype-resources/buildAndRun.sh]: Oh yeah you are right, strange that this worked yesterday 🙈

**Review by rieckpil — APPROVED** [review]
thanks for the quick fix

**Pull request merged**